masterlist.yaml

```diff
@@ -18223,6 +18223,11 @@ plugins:
         subs: [ 'imp_helm_legend_WACCF_AMB_Patch.esp' ]
         condition: 'active("Weapons Armor Clothing & Clutter Fixes.esp") and active("aMidianBorn_ContentAddon.esp")'
 
+  - name: 'Khajiit Speak - .*\.esp'
```

Comment:
Links for those patches.
https://www.nexusmods.com/skyrimspecialedition/mods/32356/
https://www.nexusmods.com/skyrimspecialedition/mods/36488/
https://www.nexusmods.com/skyrimspecialedition/mods/37024/